### PR TITLE
Replaced TSLint by ESlint

### PR DESCRIPTION
**Pull request by ceisele-r**
Replaced deprecated TSLint by its successor, ESLint

### CODE REVIEW

File: src/legacyapi/apirequests.ts [hunk 1]
```diff
@@ -32,35 +32,39 @@ export async function getJson<Request extends BaseRequest>(path: string, request
...
-      case 403:  // API_FORBIDDEN -> server requests redirect to signin
-        if (typeof window !== "undefined"  //  not possible on server rendering
+      }
+      case 403: {  // API_FORBIDDEN -> server requests redirect to signin
+        if (typeof window !== "undefined"  //  Not possible on server rendering
           && !window.location.pathname.startsWith("/signin")) {
           window.location.href = "/signin?redirect=" + encodeURIComponent(window.location.pathname);
         }
-      default:
+        break;
```
Comment (ceisele-r) [inline src/legacyapi/apirequests.ts]: I am not sure about this - was this break intentionally missing as fall-through?

File: src/legacyapi/apirequests.ts [hunk 2]
```diff
@@ -156,28 +162,32 @@ export async function getExternalJson<Request extends BaseRequest>(apiEndpointUr
...
-    case 403:  // API_FORBIDDEN -> server requests redirect to signin
-      if (typeof window !== "undefined"  //  not possible on server rendering
+    }
+    case 403: {  // API_FORBIDDEN -> server requests redirect to signin
+      if (typeof window !== "undefined"  //  Not possible on server rendering
         && !window.location.pathname.startsWith("/signin")) {
         window.location.href = "/signin?redirect=" + encodeURIComponent(window.location.pathname);
       }
-    default:
+      break;
```
Comment (ceisele-r) [inline src/legacyapi/apirequests.ts]: I am not sure about this - was this break intentionally missing as fall-through?

File: src/notification/subscriptionhandler.ts [hunk 3]
```diff
@@ -35,7 +35,7 @@ export async function initNotificationClient(user: UserDetails) {
   notificationClient = new Nes.Client(wsUrl);
   return new Promise<void>(async (resolve, reject) => {
```
Comment (ceisele-r) [inline src/notification/subscriptionhandler.ts]: Can we get rid of this Promise?

File: src/process/bpmn/bpmnextensions.ts [hunk 4]
```diff
@@ -72,6 +72,7 @@ export function getExtensionValues(activityObject: Bpmn.Activity): TaskExtension
             break;
           case "roleowners-editable":
             returnValue.roleOwnersEditable = child.$body !== "false";
+            break;
```
[the remark this answers is not in the document]
Comment (BCaspari) [reply src/process/bpmn/bpmnextensions.ts]: Looks like this was a bug

**Pull request merged**